doc/yaml_config.md

```diff
@@ -28,6 +28,40 @@ Software manager related options. It is map with following keys:
 
 Array of url for installation repositories.
 
+#### mandatory\_patterns
+
+Array of patterns that have to be selected.
+
+#### optional\_patterns
+
+Array of patterns that should be selected but can be deselected or skipped if not available.
+
+### security
+
+Options related to security
+
+#### lsm
+
+Default linux security module. Currently supported values are `selinux`, `apparmor` and `none`.
+
+#### available\_lsms
+
+Map for available linux security modules. If only one module is
+available it means that lsm is not configurable.
+
+##### patterns
```

Comment:
Just a side note not meant as a review of this: I was a big fan of the idea of having just one configuration file to shape the installation process instead of separate concepts for AutoYaST profile and control.xml. But these kind of thinks make me wonder.

Reply:
To be honest me too. I think it is kind of two separate things:

1. product specific settings, some are optional, but some are mandatory
2. user choices, which is basically capture of user intentions. Can be direct ones like CLI calls with specific value or meta ones like tuning storage.